### ISSUE TITLE
Probelm of the discriminator loss.

### ISSUE THREAD

**Issue by DesenHuang** [LiDan456/Multivariate-Anomaly-Detection-for-Time-Series-Data-with-GANs#5]
Is the discriminator loss wrong?
"D_loss_real = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=D_logit_real, labels=tf.ones_like(D_logit_real)), 1)
    D_loss_fake = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=D_logit_fake, labels=tf.zeros_like(D_logit_fake)), 1)" in model.py

I think the D_loss_fake  should be "D_loss_fake = tf.reduce_mean(tf.nn.sigmoid_cross_entropy_with_logits(logits=D_logit_fake, labels=tf.zeros_like(D_logit_fake)), 0)"

**Comment by DhrubaAdhikary** [comment]
No the Losses are correctly defined , the numeral 1 denotes the axis along which the mean needs to be reduced . and the labels are defined properly for the fake and the real .

**Comment by DesenHuang** [comment]
Oh, yep. I was wrong. Thank you very much.
